recipes/aws-noaa-whoi/recipe.py

```diff
@@ -0,0 +1,20 @@
+import s3fs
+
+from pangeo_forge_recipes.patterns import pattern_from_file_sequence
+from pangeo_forge_recipes.recipes.reference_hdf_zarr import HDFReferenceRecipe
+
+url_base = 's3://noaa-cdr-sea-surface-temp-whoi-pds/data/'
+
+years = range(1988, 2022)
```

Comment:
we might want to handle years differently, given that when data is available in 2023, this won't find the 2023 data.

years could instead be defined like so?

```suggestion
years_folders = fs.ls(join(url_base))
years = list(map(lambda x: os.path.basename(x), years_folders))
```

Reply:
good suggestion @rbavery just tested this to verify, great way to consider ingesting future years. I'll add this to the revisions